boxes.d/00-suse.yaml

```diff
@@ -0,0 +1,12 @@
+# using elastic's image since it is sp3, which is not offered by SUSE yet
+sles12sp3:
+  box_name: elastic/sles-12-x86_64
+  pty: true
+
+sles12sp2:
+  box_name: suse/sles12sp2
+  pty: true
+
+sles12sp1:
+  box_name: suse/sles12sp1
+  pty: true
```

Comment:
do these sles boxes actually work?  :/  

I'm getting:

```
URL: ["https://atlas.hashicorp.com/suse/sles12sp2"]
Error: The requested URL returned error: 404 Not Found
```

I get the same thing with the sles11 box i pointed to as well :(

Reply:
the slese12sp2 box spins up fine for me, but vagrant has already downloaded the image so I can't verify that the image downloads ok. I also tried `rm -rf /var/lib/libvirt/images/suse-VAGRANTSLASH-sles12sp2_vagrant_box_image_0.0.1.img` and respun up the box, that worked fine as well. Maybe there were network issues at the time? If you still hit it we can have someone verify (who hasn't spun up the box before)

re SLES11 box, I [commented here](https://github.com/theforeman/forklift/pull/640#issuecomment-371508641) about some issues I ran into spinning it up, so I left that one out of the PR

Reply:
@jlsherrill were you able to spin up the sles12 boxes?